pkg/logutil/log.go

```diff
@@ -292,3 +293,39 @@ func LogPanic() {
 		zaplog.Fatal("panic", zap.Reflect("recover", e))
 	}
 }
+
+var (
+	enabledRedactLog atomic.Value
+)
+
+func init() {
+	SetRedactLog(false)
+}
+
+// IsRedactLogEnabled indicates whether the log desensitization is enabled
+func IsRedactLogEnabled() bool {
+	return enabledRedactLog.Load().(bool)
+}
+
+// SetRedactLog sets enabledRedactLog
+func SetRedactLog(enabled bool) {
+	enabledRedactLog.Store(enabled)
+}
+
+// RedactArgIfNeeded will omit the argument if RedactLog is enabled
+func RedactArgIfNeeded(arg interface{}) interface{} {
```

Comment:
I think it would be more easy to use if we can provide 2 functions for `string` and `[]byte` separately.